File: internal/domain/category.go
package domain

import "time"

// ExpenseCategory represents a category for classifying expenses.
type ExpenseCategory struct {
	ID        int64
	Key       string
	LabelCS   string
	LabelEN   string
	Color     string
	SortOrder int
	IsDefault bool
	CreatedAt time.Time
	DeletedAt *time.Time
}

File: rust/crates/zfaktury-domain/src/category.rs
use chrono::NaiveDateTime;

/// A category for classifying expenses.
#[derive(Debug, Clone)]
pub struct ExpenseCategory {
    pub id: i64,
    pub key: String,
    pub label_cs: String,
    pub label_en: String,
    pub color: String,
    pub sort_order: i32,
    pub is_default: bool,
    pub created_at: NaiveDateTime,
    pub deleted_at: Option<NaiveDateTime>,
}
